### PR TITLE
DOC-7437 PR #99719 - ui: use app name of selected row when fetching fingerprint details

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-7437

For Statements and Transactions pages, updated include files to indicate that corresponding fingerprint details page reflects specified application name and database.

### CODE REVIEW

**Review by xinhaoz — APPROVED** [review]

File: src/current/_includes/v23.1/ui/statement-details.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
Comment (kathancox) [inline src/current/_includes/v23.1/ui/statement-details.md:2]: This is not part of this PR, but if so inclined, there is a rendering issue on the Statements page under "Insights" where the markdown bullets are not rendering as bullet points.

**Review by kathancox — APPROVED** [review]
LGTM, one small, non-blocking nit. And, something else that is not really part of the PR, but you might want to take a look.

File: src/current/_includes/v23.1/ui/statements-table.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ The Statements table gives details for each SQL statement fingerprint:
 
 Column | Description
 -----|------------
-Statements | SQL statement [fingerprint](#sql-statement-fingerprints). To view additional details, click the SQL statement fingerprint to open its [Statement Fingerprint page]({{ page_prefix }}statements-page.html#statement-fingerprint-page).
+Statements | SQL statement [fingerprint](#sql-statement-fingerprints). To view additional details, click the SQL statement fingerprint to open its [Statement Fingerprint page]({{ page_prefix }}statements-page.html#statement-fingerprint-page).<br /><br />Statement fingerprints are displayed per application rather than grouped into a single fingerprint. This may result in multiple rows for the same statement fingerprint, one per application name.
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/_includes/v23.1/ui/statements-table.md]: updated to make consistent.

**Review by florence-crl — COMMENTED** [review]
TFTR!
